apps/ledger-live-mobile/src/components/Stake/index.tsx

```diff
@@ -11,17 +11,27 @@ import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
 import { NavigatorName, ScreenName } from "../../const";
 import perFamilyAccountActions from "../../generated/accountActions";
 import logger from "../../logger";
+import type {
+  StackNavigatorProps,
+  BaseComposite,
+} from "../RootNavigator/types/helpers";
+import type { StakeNavigatorParamList } from "../RootNavigator/types/StakeNavigator";
 
-const StakeFlow = () => {
+type Props = BaseComposite<
+  StackNavigatorProps<StakeNavigatorParamList, ScreenName.Stake>
+>;
+
+const StakeFlow = ({ route }: Props) => {
+  const currencies = route?.params?.currencies;
   const featureFlag = useFeature("stakePrograms");
   const list = featureFlag?.params?.list;
```

Comment:
small suggestion, you could keep the previous list as it was used but instead you do here the logic of `route?.params?.currencies || featureFlag?.params?.list`, this would prevent having to send the 2 parameters in the dependency of useMemo and all the rest of the code that would use "list" would also be consistent 🤔